README.md

```diff
@@ -58,8 +58,8 @@ Vue.js is an MIT-licensed open source project. Its ongoing development is made p
 
 Vue.js is a library for building interactive web interfaces. It provides data-reactive components with a simple and flexible API. Core features include:
 
-- [Dead simple, unobtrusive reactivity using plain JavaScript objects.](http://vuejs.org/guide/overview.html#Reactive-Data-Binding)
-- [Component-oriented development style with tooling support](http://vuejs.org/guide/overview.html#Component-System)
+- [Dead simple, unobtrusive reactivity using plain JavaScript objects.](http://vuejs.org/guide/index.html#Declarative-Rendering)
```

Comment:
Agreed, let's reword this to "Declarative rendering with a plain JavaScript object based reactivity system."